### PR TITLE
BaSP-TG-73: Time-Sheet Form

### PR DESCRIPTION
**Pull request by martinlotufo93**
Shared components added. I still have to add the css styles!

### CODE REVIEW

**Comment by AlejandroSantini** [comment]
The name of the pr should respect the format 'BaSP-TG-X: Short description'

**Review by Facund0Fernandez — COMMENTED** [review]
The edit button of the list does not show the form.

File: src/Components/TimeSheet/index.js [hunk 1]
```diff
@@ -145,125 +148,77 @@ function TimeSheetForm() {
...
-          <div className={styles.modalContainer}>
-            <div className={styles.modal}>
-              <h3>{modalMessage}</h3>
-              <button disabled={isLoading} onClick={closeModal} className={styles.button}>
-                OK
-              </button>
-            </div>
+        <Button text="Submit" handler={onSubmit} />
+        <Modal isOpen={showModal} modalTitle={modalMessage} handleClose={closeModal}>
+          <div>
```
Comment (Facund0Fernandez) [inline src/Components/TimeSheet/index.js:218]: A children should be added with a data message.

File: src/Components/TimeSheet/index.js [hunk 2]
```diff
@@ -145,125 +148,77 @@ function TimeSheetForm() {
...
+            <Button text="Accept" handler={routeChange} />
```
Comment (CarVigil) [inline src/Components/TimeSheet/index.js:219]: If there is an error, this button should not redirect

**Review by CarVigil — CHANGES_REQUESTED** [review]

File: src/Components/TimeSheet/index.js [hunk 3]
```diff
@@ -1,5 +1,12 @@
 import { useState, useEffect } from 'react';
 import styles from './form.module.css';
+import Modal from '../Shared/Modal';
+import Button from '../Shared/Button';
+import { useHistory } from 'react-router-dom';
+import SelectDropdown from '../Shared/SelectDropdown';
+import Input from '../Shared/Input';
+import { Children } from 'react';
+/*import { Children } from 'react';*/
```
Comment (GuidoCerioni) [inline src/Components/TimeSheet/index.js:9]: Remove this comment

File: src/Components/TimeSheet/index.js [hunk 4]
```diff
@@ -145,125 +150,91 @@ function TimeSheetForm() {
...
-        {showModal && (
-          <div className={styles.modalContainer}>
-            <div className={styles.modal}>
-              <h3>{modalMessage}</h3>
-              <button disabled={isLoading} onClick={closeModal} className={styles.button}>
-                OK
-              </button>
-            </div>
+        <div className={styles.buttomDiv}>
+          <Button text="Update" handler={onSubmit} />
```
Comment (GuidoCerioni) [inline src/Components/TimeSheet/index.js:219]: this button should display "update" or "add" depending of the request type you are making.
Something like <Button text={requestType === 'POST' ? 'Save Timesheet' : 'Update Timesheet'} handler={onSubmit} />

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
See comments. Also you should change the PR title to "BaSP-TG-73"

**Comment by AlejandroSantini** [comment]
The branch have conflicts

**Review by Gearyandres94 — APPROVED** [review]

**Review by GuidoCerioni — APPROVED** [review]

**Review by CarVigil — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Pull request merged**